schemas/context/segmentdefinition-expression.example.1.json

```diff
@@ -0,0 +1,6 @@
+{
+    "xdm:type":"PQL",
```

Comment:
this seems like a completely new type that is Adobe specific? Is there a reason why it is not namespaced?

Reply:
XDM fields should have strong semantic meaning. `xdm:type` is already in use in the Experience Event schema. Do you feel your usage aligns with the semantic meaning of the field as it is used in the Experience Event schema?

Reply:
Hey @cfraser , you mean this - https://github.com/adobe/xdm/blob/master/schemas/data/time-series.schema.json#L31 ? I can change it to expressionType to make it specific to expressions.